### PR TITLE
Feature/sep6 17 create endpoints for reviews

### PR DESCRIPTION
**Pull request by tomondre**

### CODE REVIEW

**Review by KhaledHammoun — COMMENTED** [review]
pull the master in and refactor the code using lombok

**Comment by tomondre** [comment]
ID added, but will have to test the review flow

File: backend/src/main/java/com/sep6/backend/models/Review.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.validator.constraints.Range;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Data
```
Comment (KhaledHammoun) [inline backend/src/main/java/com/sep6/backend/models/Review.java:11]: @AllArgsConstructor
@NoArgsConstructor
I thin you need those as well, maybe the builder if you have use case for it

File: backend/src/main/java/com/sep6/backend/service/ReviewsServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+@Service
+public class ReviewsServiceImpl implements ReviewsService {
+    private ReviewsRepository repository;
+    private MoviesRepository moviesRepository;
+
+    public ReviewsServiceImpl(ReviewsRepository repository, MoviesRepository moviesRepository) {
+        this.repository = repository;
+        this.moviesRepository = moviesRepository;
+    }
```
Comment (KhaledHammoun) [inline backend/src/main/java/com/sep6/backend/service/ReviewsServiceImpl.java:19]: Lombok
@AllArgsConstructor

**Review by KhaledHammoun — CHANGES_REQUESTED** [review]

**Pull request merged**